src/azure-cli/azure/cli/command_modules/util/_help.py

```diff
@@ -21,16 +21,16 @@
 examples:
   - name: Get Audit log through Microsoft Graph
     text: >
-        az rest --method get --uri https://graph.microsoft.com/beta/auditLogs/directoryAudits
+        az rest --method get --url https://graph.microsoft.com/beta/auditLogs/directoryAudits
   - name: Update a Azure Active Directory Graph User's display name
     text: >
-        az rest --method patch --uri "https://graph.microsoft.com/v1.0/users/johndoe@azuresdkteam.onmicrosoft.com" --body "{\\"displayName\\": \\"jondoe2\\"}"
+        az rest --method patch --url "https://graph.microsoft.com/v1.0/users/johndoe@azuresdkteam.onmicrosoft.com" --body "{\\"displayName\\": \\"jondoe2\\"}"
   - name: Get a virtual machine
     text: >
         az rest --method get --uri /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}?api-version=2019-03-01
   - name: Create a public IP address from body.json file
     text: >
-        az rest --method put --uri https://management.azure.com/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/publicIPAddresses/{publicIpAddressName}?api-version=2019-09-01 --body @body.json
+        az rest --method put --url https://management.azure.com/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/publicIPAddresses/{publicIpAddressName}?api-version=2019-09-01 --body @body.json
 """
 
```

Comment:
> --uri /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}?api-version=2019-03-01
> --url https://management.azure.com/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/publicIPAddresses/{publicIpAddressName}?api-version=2019-09-01

So our current specification is that if it's a URL parameter we should use `--url`, and if it's not a URL parameter we should use `--uri`, right?